addon/components/au-toaster.hbs

```diff
@@ -0,0 +1,17 @@
+{{#if this.toaster.toasts.length}}
+<div class="au-c-toaster {{this.position}}">
+  {{#each this.toaster.toasts as |toast|}}
+    <AuAlert
+      @title={{toast.title}}
+      @skin={{toast.options.type}}
+      @icon={{toast.options.icon}}
+      @size="small"
+      @closable={{toast.options.closable}}
```

Comment:
Would it make sense to make this default to `true`? I think that's the behavior we would want in most of the use-cases?